packages/async/README.md

```diff
@@ -4,6 +4,4 @@
 	</p>
 </div>
 
-## DiscordLuau - Vendor
-
-With the lack of a Package Manager for Luau, we have a separate package for vendoring dependencies used in DiscordLuau.
+## DiscordLuau - Async
```

Comment:
> This current "async" implementation is an odd and non-standard wannabe async primitive

bro leave my smol async lib alone :cry:  

---

yes but I want to hold off on doing anything with the async stuff for now, I don't want to add more to this PR since it's already huge and instead i'll create a second PR which either removes the async lib, or refactores it to take advantage of future.

Reply:
This should be a priority after this PR is merged in that case.

Reply:
I will first test the newly added APIs, then I can work on this :+1: